src/main/java/uk/co/ramp/covid/simulation/place/Household.java

```diff
@@ -45,6 +45,7 @@ public Household(HouseholdType hType, Places places) {
         this.hType = hType;
         this.neighbours = new ArrayList<>();
         this.places = places;
+        transAdjustment = Double.MAX_VALUE;
```

Comment:
Maybe we should have a flag "noTransAdjustment" (or similar) instead of forcing a value here for clarity (I assume the MAX_VALUE is to make some test always true/false)?

Reply:
Yes. This is in there to ensure that from Place.getTransProb() we never adjust the transProb for Housheolds. I can't think of a more elegant way to do this. Suggestions are very welcome

Reply:
I think what we might want to do here is make "getTransProb" protected in Place and then use an Override in Household to return the right transition probability? Does this make sense? If not, we can merge this now and since it's correct, I can do this in some refactoring at a later point.

Reply:
I understand what you mean. I will merge this as is. There are other changes that I need to push under a separate PR, and I am concerned about the version that I am running getting upside down